lib/MaxMind/DB/Writer/Tree.pm

```diff
@@ -210,17 +213,37 @@ sub insert_range {
     my $data             = shift;
     my $additional_args  = shift // {};
 
+    $self->_validate_insertion_args($additional_args);
```

Comment:
How about rewriting this to `my $additonal_args = @_ ? $self->_validate_insertion_args( shift() ) : {};` so it's a lot cheaper in the common case when we're not passing additional args?